tools/interop-v2-reference.py

```
@@ -0,0 +1,802 @@
+        marker = self._take(1)[0]
+        if marker <= 0x7F:
+            return marker
+        if marker == 0xCC:
+            return self._take(1)[0]
+        if marker == 0xCD:
+            return int.from_bytes(self._take(2), "big")
+        if marker == 0xCE:
+            return int.from_bytes(self._take(4), "big")
+        if marker == 0xCF:
+            return int.from_bytes(self._take(8), "big")
+        raise V2Error(f"expected unsigned-family msgpack int marker, got 0x{marker:02x}")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)